src/worker/views/frivolitiesTeamSeasons.ts

```diff
@@ -315,6 +316,46 @@ const updateFrivolitiesTeamSeasons = async (
 				["most.value", "mov"],
 				["desc", "asc"],
 			];
+		} else if (type === "old_champ") {
+			title = "Oldest Championship Teams";
+			description = "These are oldest teams that won the title.";
+			extraCols.push(
+				{
+					key: "avgAge",
+					colName: "AvgAge",
+				},
+				{
+					key: "seed",
+					colName: "Seed",
+				},
+				{
+					key: ["most", "roundsWonText"],
+					keySort: "playoffRoundsWon",
+					colName: "Rounds Won",
+				},
+			);
+
+			filter = ts =>
+				!!ts.avgAge &&
```

Comment:
It doesn't actually matter here because avgAge will never be 0, but... well technically it could, people can use God Mode to make negative age players. And we want this feature to work then, right? So don't use `!!` to check for undefined, because it really checks for anything falsy, including 0. Instead do `ts.avgAge !== undefined`.